### ISSUE TITLE
Modifiers _capture doesn't work with el-col of ElementUi

### ISSUE THREAD

**Issue by HawtinZeng** [vuejs/jsx#309]
![image](https://github.com/vuejs/jsx-vue2/assets/73981739/00c344ab-49c0-497a-a20c-dabd7247672f)
I want to do the outer click event handler before the inner click, so I use _capture modifiers, but I found `vOn_click_native_capture` can' trigger the click event, what should I do??
The online playground is here: https://stackblitz.com/edit/vue2-vue-cli-ewhdaa?file=src%2Fcomponents%2FOuter.vue

**Comment by HawtinZeng** [comment]
The online playground is here, https://stackblitz.com/edit/vue2-vue-cli-ewhdaa?file=src%2Fcomponents%2FOuter.vue

**Comment by Sun79** [comment]
You can temporarily work around this bug using the following code:
```js
<el-col {...{ nativeOn: { '!click': () => {} } }}></el-col>
```
Or wait for this pull request to be merged: #303.

**Comment by HawtinZeng** [comment]
Ok, thank u.
